modules/ruxhal/src/arch/riscv/trap.rs

```diff
@@ -29,8 +29,23 @@ fn riscv_trap_handler(tf: &mut TrapFrame, _from_user: bool) {
     match scause.cause() {
         Trap::Exception(E::Breakpoint) => handle_breakpoint(&mut tf.sepc),
         Trap::Interrupt(_) => crate::trap::handle_irq_extern(scause.bits()),
+        #[cfg(feature = "musl")]
+        Trap::Exception(E::UserEnvCall) => {
+            let ret = crate::trap::handle_syscall(
+                tf.regs.a7,
+                [
+                    tf.regs.a0 as _,
+                    tf.regs.a1 as _,
+                    tf.regs.a2 as _,
+                    tf.regs.a3 as _,
+                    tf.regs.a4 as _,
+                    tf.regs.a5 as _,
+                ],
+            );
+            tf.regs.a0 = ret as _;
+        }
         _ => {
-            panic!(
+            error!(
```

Comment:
It is better to just let it panic instead of throwing out an error.

Reply:
Sure. I forgot to change it back after debugging.